spec/factories/providers.rb

```diff
@@ -1,6 +1,7 @@
 FactoryBot.define do
   factory :provider do
     firm
+    id { SecureRandom.uuid }
```

Comment:
That shouldn't be necessary. Why did you need to do this? The database should be generating an `id` on create. If it isn't in your tests - that suggest something isn't right.